ea_server/api/utils.py

```diff
@@ -0,0 +1,59 @@
+import functools
+import math
+def transform(result, data):
+    result_as_dict = dict()
+    for order_index,p in enumerate(result):
+        driver_index = math.floor(p)
+        driver_id = data["drivers"][driver_index]["id"]
+        order_in_route = p-driver_index
+        item=(data["orders"][order_index]["id"],p)
+        
+        if data["drivers"][driver_index]["id"] in result_as_dict:
```

Comment:
I think that we can define a dictionary with all the drivers at first with empty list and avoid this conditions.

Also, if we'll sort the list before loop over it we can avoid the maintenance of the ordered lists and just append the orders one by one.